extensions/cocostudio/WidgetReader/ArmatureNodeReader/ArmatureNodeReader.cpp

```diff
@@ -7,9 +7,9 @@
 #include "CSParseBinary_generated.h"
 #include "WidgetReader/ArmatureNodeReader/CSArmatureNode_generated.h"
 #include "Armature.h"
-#if defined(AX_ENABLE_EXT_DRAGONBONES)
-#    include "DragonBones/CCDragonBonesHeaders.h"
-#endif
+//#if defined(AX_ENABLE_EXT_DRAGONBONES)
```

Comment:
remove these lines instead of commend since CCDragonBonesHeaders.h was removed

Reply:
Its in draft ...its more to do.